tox.ini

```diff
@@ -7,18 +7,16 @@
 
 [tox]
 envlist = py27-swift{1.13.1,2.1.0,2.2.0,2.3.0,2.5.0,2.7.1,2.10.1,2.13.1,2.15.1}
-          py27-swifthead
```

Comment:
Any way to keep `head` working? Maybe set it as 'may fail'?

Reply:
Hm as far as I can see in the tox documentation, it seems that a command can be set as 'may fail' by appending a dash (-) before it, but I haven't found information related to envlist elements that can be set the same way.

As it will be restored in the PR merging the queens_compat branch, maybe I can just comment the swifthead lines instead of removing them I you wish?